tools/SourceKit/include/SourceKit/Core/LangSupport.h

```diff
@@ -730,9 +741,8 @@ class LangSupport {
   virtual void
   getCursorInfo(StringRef Filename, unsigned Offset, unsigned Length,
                 bool Actionables, bool CancelOnSubsequentRequest,
-                ArrayRef<const char *> Args,
-                llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> FileSystem,
-                std::function<void(const RequestResult<SourceKit::CursorInfoData> &)> Receiver) = 0;
+                ArrayRef<const char *> Args, Optional<VFSOptions> vfsOptions,
+       std::function<void(const RequestResult<CursorInfoData> &)> Receiver) = 0;
```

Comment:
Formatting looks a bit weird here.

Reply:
I think this is an intentional thing that is done some places in the Swift codebase to keep lines under the column limit. Anyways, this is from upstream so I won't make any changes.